src/Sarif.Driver.UnitTests/Sdk/FormatForVisualStudioTests.cs

```diff
@@ -15,7 +15,6 @@ public class FormatForVisualStudioTests
         private const string TestRuleId = "TST0001";
         private const string TestFormatSpecifier = "testFormatSpecifier";
         private const string TestAnalysisTarget = @"C:\dir\file";
-        private static readonly string DisplayedTarget = TestAnalysisTarget.Replace('\\', '/');
```

Comment:
@lgolding, note I fixed the forward slashes that appeared in the VS output (which was certainly not by design).